src/sqlancer/mysql/MySQLProvider.java

```diff
@@ -153,6 +156,31 @@ public void generateDatabase(MySQLGlobalState globalState) throws Exception {
                     }
                 });
         se.executeStatements();
+
+        if (globalState.getDbmsSpecificOptions().getTestOracleFactory().size() == 1
+                && globalState.getDbmsSpecificOptions().getTestOracleFactory().get(0)
+                        .create(globalState) instanceof MySQLCERTOracle) {
+            // Enfore statistic collected for all tables
+            ExpectedErrors errors = new ExpectedErrors();
+            MySQLErrors.addExpressionErrors(errors);
+            for (MySQLTable table : globalState.getSchema().getDatabaseTables()) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("ANALYZE TABLE ");
+                sb.append(table.getName());
+                sb.append(" UPDATE HISTOGRAM ON ");
+                boolean first = true;
+                for (MySQLColumn c : table.getColumns()) {
```

Comment:
I think this loop could be a one-liner with the streaming API? See https://stackoverflow.com/questions/37987051/how-to-concatenate-a-string-with-the-new-1-8-stream-api.